src/main/kotlin/no/nav/syfo/behandlendeenhet/api/internad/BehandlendeEnhetAPI.kt

```diff
@@ -95,5 +100,50 @@ fun Route.registrerPersonApi(
                 call.respond(HttpStatusCode.BadRequest)
             }
         }
+
+        post("/personer") {
```

Comment:
Vanskelig å vite hva denne best kan hete for å gi mening 🤔 En `POST` til et `/personer` endepunkt hadde jeg tenkt opprettet en ny personressurs. 
Hva med en `PATCH` på `/oppfolgingsenhet-tildelinger` eller `/behandlendeenhet-tildelinger`? 🤔 `behandlendeenhet` er kanskje hakket mer riktig ettersom man kan bruke det til å sette geografisk. Var ikke kjempe rett frem denne her, hva tenker du? 😊u

Reply:
Ja, i noen tilfeller er det jo en post, i andre er det en patch 🤷🏼‍♂️ Men jeg tenker post er enklest da 🤔
Enig i at navnet på ressursen burde være noe annet enn `/personer` 👍🏼